src/components/Worker.tsx

```diff
@@ -0,0 +1,111 @@
+interface WorkerProps {workerInfo: {id: number, firstName: string, lastName:string,
+     title:string, country: string, city:string, birthDate:Date, imageUrl:string, time: string}}
+
+const Worker = ({workerInfo}: WorkerProps) => {
+    const age: number = new Date().getFullYear() - workerInfo.birthDate.getFullYear();
+    console.log(workerInfo.birthDate);
+    return (
+        <>
+            <div className="col-md-4">
```

Comment:
Add to VS code an extention called prettier and add format on save option to make this code easier to read